synse_server/metrics.py

```diff
@@ -134,11 +132,11 @@ def register(self) -> None:
 
         @self.app.middleware('request')
         async def before_request(request: Request) -> None:
-            request[self._req_start_time] = time.time()
+            request.ctx.req_start_time = time.time()
 
         @self.app.middleware('response')
         async def before_response(request: Request, response: HTTPResponse) -> None:
-            latency = time.time() - request[self._req_start_time]
+            latency = time.time() - request.ctx.req_start_time
```

Comment:
This looks familiar :thinking:  did we do the same update for network-api earlier this week?

Reply:
yup. the change here is pre-emptive since it was using an older version of sanic, but I figured if we knew about the breaking  change, it makes sense to update the dependency and fix  it so its not an issue later on.